queries.sql

```diff
@@ -15,6 +15,8 @@
 -- thinking the better feedback we'll be able to give!
 
 -- "albums" table
+SELECT albums;
```

Comment:
Hah, that's not meant to be a query. It's meant to signal the start of a section involving the albums table.